docs/workflow/pycbc_make_coinc_search_workflow.rst

```diff
@@ -608,53 +608,250 @@ If you have a subdax that failed, ``pegasus_analyzer`` will provide you with a c
+* The previous workflow completed successfully.
+
+* There are no changes to the workflow configuration file, other than incrementing the end time of the workflow.
+
+In this case, first re-run ``pycbc_make_coinc_search_workflow`` to build the
+new workflow, and then create a cache file in the following way:
+
+1. Locate the PyCBC result page for the workflow that you wish to extend.
+
+2. In the menu under **Section 8: Workflow**, locate the **Output map** section (usually Section 8.06) and open that page.
+
+3. This page will show three output cache files that contain the URLs of the data created by the workflow. Locate the file that ends ``main.map`` and download it by clicking on the **Link to file**. This file contains the main intermediate and output data products of the workflow.
+
+4. Edit this file so that it only contains the output of the ``pycbc_inspiral`` jobs, i.e. delete all of the lines that do not match the pattern ``*INSPIRAL*hdf``. You can do this in a text editor, or with your favorite combination of UNIX ``grep``, ``sed``, ``awk``, or ``perl`` commands.
+For example::
+
+    egrep 'INSPIRAL.*hdf' /path/to/downloaded/workflow-main.map > inspiral_files.map
+
+will pull out all cache file lines for the outputs of ``pycbc_inspiral`` files and write them to a new cache file called ``inspiral_files.map``.  
 
----------------------------
-Plan the workflow
----------------------------
+5. If the files in the new cache file exist locally on the cluster where you are submitting the workflow, then the cache file is complete. If they do not, you will need to modify the file to change the ``PHYSICAL_FILE_URL`` to a valid ``gsiftp://`` or ``http://`` URL on the remote cluster, and change ``pool="local"`` to ``pool="remote"``. Again, these changes can be made with a text editor or UNIX shell tools. For example, if the file URLs begin with ``/home/dbrown`` and they are on the Syracuse cluster, to run on Atlas you would use the following ``sed`` commands to change the ``SITE`` and the URI in the cache file::
 
-From the directory where the dax was created, now plan the workflow with an additional argument as follows.::
+    sed 's/pool="local"/pool="remote"/g' inspiral_files.map > inspiral_files.map.tmp
+    sed 's+/home/dbrown+gsiftp://sugwg-condor.phy.syr.edu/home/dbrown+g' inspiral_files.map.tmp > inspiral_files.map
+    rm inspiral_files.map.tmp
 
-    pycbc_submit_dax --accounting-group ligo.dev.o1.cbc.explore.test --dax s6d_chunk3.dax --cache /path/to/prior_data.map
+6. Finally, copy the file ``inspiral_files.map`` to your new workflow directory and then run ``pycbc_submit_dax`` as usual, giving the path to ``inspiral_files.map`` as the ``--cache-file`` argument.
+
+---------------------------------------------------
+Re-running a workflow using a new veto definer file
+---------------------------------------------------
+
+Data reuse can be used to re-running a workflow with a new veto definer file, assuming that:
```

Comment:
it's not clear to me that the calculate_psd and optimal_snr jobs will always be valid after the veto definer is changed - in principle if there are new significant CAT1 vetoes the psds could have substantial changes.  if there are only CAT2+ veto changes then they are OK (and so are the hdf_trigger_merge files ..)